R/rd-html.R

```diff
@@ -100,7 +109,17 @@ as_html.tag_subsection <- function(x, ...) {
 
 #' @export
 as_html.tag_eqn <- function(x, ..., mathjax = TRUE) {
-  stopifnot(length(x) <= 2)
+  if ( length(x) > 2 ) {
+    dots <- list(...)
```

Comment:
* I think this should use `context_get("rdname")` to get the name of the input file. 

* I don't think it's worth changing the interface of the function in order to add the section name, we can tackle that in the future.

* Since you've repeated the same code multiple times, please extract it out into a function